gengokit/httptransport/templates_test.go

```diff
@@ -201,7 +197,7 @@ func DecodeHTTPSumZeroRequest(_ context.Context, r *http.Request) (interface{},
 `
 	if got, want := str, desired; got != want {
```

Comment:
To me, where this is useful is when updating tests, sometimes the log message if forgotten and so this makes it so you only change the thing you are comparing against and the log updates too. i.e.

Rather than this:

```
	if got:= svc.GetName()  got != "ProtoService";  {
		t.Fatalf("\n`%v` was service name\n`%v` was wanted", got, "ProtoSerice")
	}
```

Do this:

```
	if got, want := svc.GetName(), "ProtoService"; got != want {
		t.Fatalf("\n`%v` was service name\n`%v` was wanted", got, want)
	}
```

I can see it being extra verbose though.